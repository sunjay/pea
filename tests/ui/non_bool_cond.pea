fn main() {
    if 0 {}
}
