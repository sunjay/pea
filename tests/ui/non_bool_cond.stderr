stack backtrace:
   0: main
error: conditions must evaluate to a value of type `bool`
